practice/src/scripts/constants/notify.ts

```diff
@@ -12,63 +12,73 @@ export const Notify = {
     EDIT: "edit",
 };
 
-export const messageNotify: {
-    [x: string]: {
-        type: string;
-        message: string;
-    };
-} = {
-    [Notify.All]: {
-        type: Notify.All,
-        message: "Your action has been executed! The active tasks are showing.",
-    },
-    [Notify.ACTIVE]: {
-        type: Notify.ACTIVE,
-        message: "Your action has been executed! The active tasks are showing.",
+const MESSAGES = {
+    All : "Your action has been executed! The active tasks are showing",
+    ACTIVE: "Your action has been executed! The active tasks are showing.",
+    COMPLETED: "Your action has been executed! The completed tasks are showing",
+    ADD: "Your action has been executed! A task was added successfully.",
+    CLEAR: "Your action has been executed! All of the completed tasks were cleared.",
+    REMOVE: "Your action has been executed! A task was removed successfully.",
+    TOGGLE_ACTIVE: "Your action has been executed! A task was checked done successfully.",
+    TOGGLE_UN_ACTIVE: "Your action has been executed! A task was unchecked done successfully.",
+    TOGGLE_ACTIVE_ALL: "Your action has been executed! All of the tasks are checked as completed.",
+    TOGGLE_UN_ACTIVE_ALL: "Your action has been executed! All of the tasks are unchecked as completed.",
+    EDIT: "Your action has been executed! A task was updated successfully.",
+}
+
+export const messageNotify = {
+
+    [TYPES.All]: {
```

Comment:
[TYPES.All]: {
  type: TYPES.All,
   message: MESSAGES.All
}
if you have type = [TYPES.All] and find an object ihave type TYPES.All and the message: MESSAGES.All
so you can write shorthand
[TYPES.All]: MESSAGES.All